packages/network/src/services/http/headers.ts

```diff
@@ -54,14 +66,18 @@ export class HTTPHeaders {
         this._setHeader(key, value);
     }
 
-    toHeadersInit(): HeadersInit {
+    toHeadersInit(body?: any): HeadersInit {
         const headers: HeadersInit = {};
         this._headers.forEach((values, key) => {
             headers[key] = values.join(',');
         });
 
         headers.accept ??= 'application/json, text/plain, */*';
-        headers['content-type'] ??= 'application/json;charset=UTF-8';
+
+        // If the request body is FormData, the browser will automatically generate Content Type with boundary
+        if (!(body instanceof FormData)) {
```

Comment:
Consider if the body is `instanceof FormData`, is it better to set the `content-type` to `application/x-www-form-urlencode` for the user.

Reply:
Other PR optimizations https://github.com/dream-num/univer-pro/issues/4222